roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/util/ConfigParserTest.java

```diff
@@ -39,15 +39,16 @@ public void configParserTest_roundaboutLoaded() {
 
     @Test
     public void configParserTest_roundaboutSectionsLoaded() {
+        assertNotNull("roundabout sections loaded", roundAboutConfig.getRoundabout().getSections());
```

Comment:
what if the list is not null but has zero sections? are then really "sections loaded"?